package.json

```diff
@@ -13,16 +13,20 @@
   "files": [
     "dist"
   ],
-  "main": "./dist/index.umd.cjs",
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./dist/index.js",
-      "require": "./dist/index.umd.cjs"
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
```

Comment:
I noticed that when running `pnpm run build` we only build the ESM bundle along with types. No sign of a `umd.cjs` bundle. Is this intentional? If so, I removed the "require" and "main" export entrypoints.

Reply:
These export values were copied over from a template. As far as I know they are unused until now. 

I defer to you if we should be publishing cjs or not.